geoips/plugins/yaml/sectors/dynamic/tc_1024x1024/tc_1km_1024x1024.yaml

```diff
@@ -1,15 +1,20 @@
-interface: "sectors_dynamic"
-metadata:
-  name: "tc_1km_1024x1024"
-  family: "standard"
-  short-description: "The tc_1km_1024x1024 boundaries configuration"
-  long-description: "Using pixel_width: 1000, pixel_height: 1000, num_lines: 1024, num_samples: 1024."
+interface: sectors
+family: generated
+name: "tc_1km_1024x1024"
+docstring: |
+  The tc_1km_1024x1024 boundaries configuration.
+
+  Using pixel_width: 1000, pixel_height: 1000, num_lines: 1024, num_samples: 1024.
 spec:
-  area_def_generator:
-    name: clat_clon_resolution_shape
-    arguments:
-      projection: eqc
-      pixel_width: 1000
-      pixel_height: 1000
-      num_lines: 1024
-      num_samples: 1024
+  generators:
+    metadata:
```

Comment:
I think all we will need to do here to make it work with the current layout of geoips is remove the metadata generator section.  These are fine as is for this PR, we will finalize the dynamic template formatting in a later PR

Reply:
Sounds good. Thanks Mindy.